content/docs/introducing-jsx.md

```diff
@@ -181,4 +181,5 @@ We will explore rendering React elements to the DOM in the next section.
 
 >**Tip:**
 >
->We recommend using the ["Babel" language definition](https://babeljs.io/docs/editors) for your editor of choice so that both ES6 and JSX code is properly highlighted. This website uses the [Oceanic Next](https://github.com/voronianski/oceanic-next-color-scheme) color scheme which is compatible with it.
+
```

Comment:
```suggestion
```

Reply:
Ya removed it in https://github.com/reactjs/reactjs.org/pull/2660